.eslintrc.js

```diff
@@ -196,5 +218,40 @@ module.exports = {
       },
     ],
     'no-use-before-define': 'off',
+
+    // Typescript Rules
+    // custom rules for typescript-eslint: https://github.com/OperationCode/front-end/pull/1792#pullrequestreview-1874516174
+    '@typescript-eslint/consistent-type-imports': 'error',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/naming-convention': [
```

Comment:
these rules would need to go within overrides with the parser, no?

Reply:
That's what I thought and put it there but for some reason it wasn't working for me. Maybe my config was bad and I'll try again.